bin/build

```diff
@@ -0,0 +1,82 @@
+#!/usr/bin/env node
+
+var md = require('meta-marked'),
+    fs = require('fs'),
+    json = require('jsonfile'),
+    path = require('path');
+
+if (! fs.existsSync('meta.json')) {
+  throw new Error('meta.json does not exist!');
+}
+var meta = json.readFileSync('meta.json');
+
+var excluded_files = [
+  'CONTRIBUTING.md',
+  'README.md'
+];
+
+var output = {
+  title: meta.title,
+  route: '/',
+  articles: getArticlesFromPath('.'),
+  routes: []
+}
+
+// Check if we have any top level articles
+
+for (var cat of meta.categories) {
+  output.routes.push({
+    route: cat.link,
+    title: cat.title,
+    articles: getArticlesFromPath(cat.link)
+  });
+}
+
+json.writeFileSync('./build/data.json', output, {spaces: 2});
+
+function getArticlesFromPath(link) {
+  var articles = {};
+  if (! fs.existsSync(link)) {
```

Comment:
You use ```existsSync``` and ```lstatSync().isDirectory()``` on 58. Should probably choose one or the other for consistency.

Reply:
`lstatSync` throws an exception if the file doesnt exist. They serve different purposes.